api/v1alpha1/constants.go

```diff
@@ -22,12 +22,15 @@ const (
 
 	ManagementClusterAkoDeploymentConfig = "install-ako-for-management-cluster"
 
-	AkoUserRoleName              = "ako-essential-role"
-	ClusterFinalizer             = "ako-operator.networking.tkg.tanzu.vmware.com"
-	AkoDeploymentConfigFinalizer = "ako-operator.networking.tkg.tanzu.vmware.com"
-	AkoDeploymentConfigKind      = "AKODeploymentConfig"
-	AkoDeploymentConfigVersion   = "networking.tanzu.vmware.com/v1alpha1"
-	AkoStatefulSetName           = "ako"
+	AkoUserRoleName               = "ako-essential-role"
+	ClusterFinalizer              = "ako-operator.networking.tkg.tanzu.vmware.com"
+	AkoDeploymentConfigFinalizer  = "ako-operator.networking.tkg.tanzu.vmware.com"
+	AkoDeploymentConfigKind       = "AKODeploymentConfig"
+	AkoDeploymentConfigVersion    = "networking.tanzu.vmware.com/v1alpha1"
```

Comment:
Question: Do we keep the ADC for backward compatibility?

Reply:
Currently yes.